### ISSUE TITLE
Please publish to PyPi

### ISSUE THREAD

**Issue by amykyta3** [return42/linuxdoc#5]
Currently, linuxdoc documentation recommends installation of the extension [via the GitHub git path](https://return42.github.io/linuxdoc/install.html). Any reason why you don't publish it as a package to [PyPi](https://pypi.org/)?

I think you have an excellent set of plugins. Publishing to the Python package index would be a great way to share this work!

**Comment by return42** [comment]
Hi @amykyta3, sorry for my late reply .. no, there was no special reason. Since I do not maintain any patch level nor any subversion I recommend to install *bleeding edge* from repositories.

If you want, I can build a package for PyPi.  From your POV, what is the benefit of PyPi over a installation from the origin?

**Comment by amykyta3** [comment]
My workplace's security policies make it difficult to access public repositories directly. We can work around it by hosting a fork internally, but its not ideal. Publishing to PyPi is preferred since we have a more established process for mirroring that database.
I'm sure others have similar policies in place as well.

**Comment by return42** [comment]
Yes, DMZ .. I should know ;)

I will build a package next days/weeks ..  ATM my time is very limited / thanks for being patience with me.

**Comment by amykyta3** [comment]
No problem. Stay safe!

**Comment by return42** [comment]
@amykyta3 sorry for the very long delay .. 

TLDR: Sphinx V3.0 and v3.1 brings a complete rewrite of the c-domain which is not downward compatible. I had to handle this before release and my time was very limited last year.  Some issues (superfluous warnings) still exists.  I hope this year will be a bit more relaxed and I have more time to maintain linuxdoc .. :relaxed:

**Comment by marxin** [comment]
Hey. Just wanted to point out that I would also recommend publishing a PyPi package. Thanks.
